nixpkgs_review/github.py

```diff
@@ -114,6 +115,12 @@ def graphql(self, query: str) -> dict[str, Any]:
 
     def pull_request(self, number: int) -> Any:
         "Get a pull request"
+
+        if pr_json := os.environ.get("NIXPKGS_REVIEW_PULL_REQUEST_JSON"):
```

Comment:
I won't block on this, but it'd be nice to also have a CLI argument in addition to the environment variable.